nursery/encrypt-data-using-hc-128.yml

```diff
@@ -0,0 +1,48 @@
+rule:
+  meta:
+    name: encrypt-data-using-hc-128
+    namespace: data-manipulation/encryption/hc-128
+    author: '@recvfrom'
+    description: Looks for instruction mnemonics associated with initialization of the HC-128 stream cipher
+    scope: basic block
+    examples:
```

Comment:
can you please add this ATT&CK info here?
```yaml
att&ck:
      - Defense Evasion::Obfuscated Files or Information [T1027]
```

Reply:
Will do! I too feel pretty good about the rule after adding in the mnemonic counts.  Should the rule be moved out of the nursery, then?

Reply:
Yes, please!